app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,24 @@
+module SessionsHelper
+  def login(user)
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    if session[:user_id]
```

Comment:
I would rewrite this method a little bit:
```ruby
def current_user
  return if session[:user_id].nil? # in case the user_id is missing from session, this will return nil

  return @current_user if @current_user.present? # in case there already is a current_user instance, return it

  @current_user = User.find(session[:user_id]) # find the user & return it if got here
end
```

This is generally easier to read, as it's clear which each stop is. We first check if the session exists, or we exit the method with nil. We then check if the current_user exists and return it, or go further

Also, `.find(id_variable)` is the same as `.find_by(id: id_variable)`.